src/host-web/chucknode-postjs.js

```diff
@@ -509,6 +509,24 @@ class ChuckNode extends AudioWorkletProcessor
                 {
                     return function()
                     {
+                        // Create directories required to load filesToPreload
+                        const dirs = filesToPreload.map(
+                            (f) => {
+                                const path = f.filename.split("/");
+                                return path.slice(0, path.length - 1);
+                            }
+                        );
```

Comment:
nit: this would help for readability:

```js
const dirs = filesToPreload.map((f) => {
    const path = f.filename.split("/");
    return path.slice(0, path.length - 1);
});
```